### PR TITLE
[4] s/mycrypt/mcrypt

### PR DESCRIPTION
**Pull request by PhilETaylor**
My bad this was my typo

s/mycrypt/mcrypt

### CODE REVIEW

**Comment by richard67** [comment]
Mycrypt .. is that encryption for MySQL? 😄 (joking)

File: administrator/components/com_users/src/Model/UserModel.php [hunk 1]
```diff
@@ -1040,7 +1040,7 @@ public function getOtpConfig($userId = null)
 			 *
 			 * The aim is/was to migrate away from mcrypt encrypted data by decrypting the data and then re-encrypting
 			 * it with the openssl adapter, but there has been a bug for a long time in the constructing of the
-			 * mycrypt Aes class, where the number of parameters passed were wrong, meaning it was actually returning
+			 * mcrypt Aes class, where the number of parameters passed were wrong, meaning it was actually returning
 			 * an openssl adapter not a mycrypt one.
```
Comment (Quy) [inline administrator/components/com_users/src/Model/UserModel.php:1044]:
```suggestion
			 * an openssl adapter not a mcrypt one.
```

**Comment by zero-24** [comment]
Merging thanks @PhilETaylor

**Comment by richard67** [comment]
@zero-24 It seems you were too fast because @Quy found a few more.

**Comment by PhilETaylor** [comment]
for some reason `mycrypt` is in my custom dictionary :)

File: administrator/components/com_users/src/Model/UserModel.php (hunk 1 above)
Comment (PhilETaylor) [reply administrator/components/com_users/src/Model/UserModel.php:1044]: will pr that one as this one is merged. thanks

**Comment by richard67** [comment]
How about "Mycrosoft"?

File: administrator/components/com_users/src/Model/UserModel.php (hunk 1 above)
Comment (brianteeman) [reply administrator/components/com_users/src/Model/UserModel.php:1044]: that would mean it should change to "an ...

Comment (richard67) [reply administrator/components/com_users/src/Model/UserModel.php:1044]: That's right. @PhilETaylor Do you wanna make another PR to fix the remaining ones?

**Pull request merged**